services/ui-src/src/components/cards/EntityStepCard.test.tsx

```diff
@@ -175,12 +154,11 @@ const NoOpenAddEditEntityFunction = (
     verbiage={mockModalDrawerReportPageJson.verbiage}
     openDeleteEntityModal={openDeleteEntityModal}
     openDrawer={mockOpenDrawer}
-    printVersion={false}
   />
 );
 
-describe("<EntityCard />", () => {
-  describe("Completed EntityCard", () => {
+describe("<EntityStepCard />", () => {
+  describe("Completed EntityStepCard", () => {
```

Comment:
I'm seeing `Warning: React does not recognize the entityIndex prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase entityindex instead. If you accidentally passed it from a parent component, remove it from the DOM element.` from running tests.